src/lib/HighlightHashLink.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
can be inlined in https://github.com/demos-europe/demosplan-core/blob/20c149f737d338082b8730349da90e94a9a5dc3b/client/js/bundles/faq/faqlist.js as it is used only there.

Reply:
Done in:
https://github.com/demos-europe/demosplan-ui/pull/71/commits/d90277fdb90ee4349783e5c1e51137121a6e26c6
New PR in demosplan-core: https://github.com/demos-europe/demosplan-core/pull/666